src/Illuminate/Foundation/helpers.php

```diff
@@ -470,6 +470,27 @@ function factory()
     }
 }
 
+if (! function_exists('has_errors')) {
+    /**
+     * Returns true if session has errors, and optionally key-specific errors.
+     *
+     * @param  string|null  $key
+     * @param  string|null  $bag
+     * @return bool
+     */
+    function has_errors($key = null, $bag = 'default')
```

Comment:
@vlakoff I think I'll revert back to 'default' for consistency. You can submit a PR changing both if this gets merged.